test/README.md

```diff
@@ -0,0 +1,22 @@
+Scylla in-source tests.
+
+For details on how to run the tests, see docs/dev/testing.md
```

Comment:
You can make this a clickable link (on the Github UI) by changing it to `[docs/dev/testing.md](../docs/dev/testing.md)`.

Reply:
Done.